WordPress/Classes/Utility/Blogging Prompts/PromptRemindersScheduler.swift

```diff
@@ -43,9 +43,15 @@ class PromptRemindersScheduler {
     ///   - time: The user's preferred time to be notified.
     ///   - completion: Closure called after the process completes.
     func schedule(_ schedule: BloggingRemindersScheduler.Schedule, for blog: Blog, time: Date? = nil, completion: @escaping (Result<Void, Error>) -> Void) {
+        guard let context = blog.managedObjectContext else {
+            return
```

Comment:
Do you think we should call the `completion` before returning? It's usually good to make sure completion is called once and once only.

Reply:
Good catch! This is addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/20854/commits/241e0ed5d0430a816bf76e76798cf2698b00ccff.